src/modules/ilos/infrastructure/decorators/ilo-permission.decorator.spec.ts

```
@@ -0,0 +1,78 @@
+    };
+  }),
+}));
+
+describe('IloPermission Decorator', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should set metadata with single permission', () => {
+    const mockSetMetadata = SetMetadata as jest.Mock;
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)